### PR TITLE
Modified readme structure.

### PR DESCRIPTION
**Pull request by fgoudreault**
- Added link to read the docs

- Moved tutorial of readme inside another file and added information in readme

### CODE REVIEW

File: doc/tutorial.rst [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+Function that returns a named tuple serving as a place holder for all the data
+needed for an operation point in the report
+
+Usage
+=====
+
+To use the code, you need to have a set of images that correspond to an
+EMVA1288 test. There are some sample image sets provided by the standard
+development group. `Example
+datasets <https://emva1288.plan.io/projects/emva1288-standard-public/files>`__.
```
Comment (fariza) [inline doc/tutorial.rst]: point this to the new datasets in github

File: doc/tutorial.rst [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+--------------------------------------------------------
+
+This class takes an EMVA1288 descriptor file and loads its content into
+a python dictionary.
+
+An EMVA1288 descriptor file is a file that contains the description of
+an EMVA1288 test including exposure times, photon count and
+corresponding images relative path to the descriptor.
+
+`Example of a descriptor file <https://github.com/EMVA1288/emva1288/blob/master/examples/EMVA1288_Descriptor_File.txt>`__
```
Comment (fariza) [inline doc/tutorial.rst]: point this to one of the new datasets descriptor files

Comment (fariza) [inline doc/tutorial.rst]: remove the file from the examples

File: doc/tutorial.rst [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+    # Load your data descriptor file
+    e = Emva1288(filename)
+    # Print the results
+    e.results()
+    # Plot the graphics
+    e.plot()
+
+If you are interested keep reading.
+
+The code is divided in 7 parts
```
Comment (fariza) [inline doc/tutorial.rst]: this is not True anymore. Now we have cam and other stuff..... what to do?

File: README.rst [hunk 4]
```diff
@@ -9,110 +9,33 @@ Please visit `Emva1288
...
 
+Documentation
+=============
+
+The whole module documentation is hosted on `this website
+<http://emva1288.readthedocs.io/en/latest/`
+There, the api documentation can be found along with tutorials on how to
+use the module.
+
 Installation
```
Comment (fariza) [inline README.rst]: la procedure d'installation doit etre dans la doc, pas ici.

Dans le readme on disse c'est quoi l'EMVA1288 et pointe vers la doc

File: doc/tutorial.rst (hunk 1 above)
Comment (fgoudreault) [inline doc/tutorial.rst]: Done

File: doc/tutorial.rst (hunk 2 above)
Comment (fgoudreault) [inline doc/tutorial.rst]: Done

File: doc/tutorial.rst (hunk 3 above)
Comment (fgoudreault) [inline doc/tutorial.rst]: I suggest that it stays like this for the moment and treat the camera simulator and dataset generator as add-ons to the main process module.

File: README.rst (hunk 4 above)
Comment (fgoudreault) [inline README.rst]: Done moving installation instructions into new file.

**Pull request merged**